File: contest/hackerrank/grading_students.rb
#!/bin/ruby
require 'pp'


def roundoff(grade)
	if grade>37
		remainder = grade%5
		if remainder>3
			return (grade+(5-remainder))
		end
	end
	return grade	
end

n=gets.strip.to_i
grades = Array.new(n)
for index in (0..n-1)
	temp = gets.strip.to_i
	puts temp
end

File: contest/hackerrank/grading_students.py
#!/bin/python3
import sys
def getrounded(grade):
	if grade > 37 :
		remainder = grade%5
		if remainder >= 3 :
			return (grade + (5-remainder))
	return grade

n = int(input().strip())
i=0
for i in range(n):
	num = int(input().strip())
	print(getrounded(num))
